File: src/back-end/CryptEx/CryptExApi/Models/ViewModel/Wallets/TotalsViewModel.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace CryptExApi.Models.ViewModel.Wallets
{
    public class TotalsViewModel
    {
        public TotalViewModel AccountTotal { get; set; }

        public TotalViewModel FiatTotal { get; set; }

        public TotalViewModel CryptoTotal { get; set; }


        public TotalViewModel AccountTotalYtd { get; set; }

        public TotalViewModel FiatTotalYtd { get; set; }

        public TotalViewModel CryptoTotalYtd { get; set; }


        public TotalViewModel PerformanceYtd { get; set; }
    }
}

File: src/front-end/CryptExFrontEnd/src/app/wallet/models/totals-view-model.ts
import { TotalViewModel } from "./total-view-model";

export interface TotalsViewModel {
    accountTotal: TotalViewModel;
    fiatTotal: TotalViewModel;
    cryptoTotal: TotalViewModel;

    accountTotalYtd: TotalViewModel;
    fiatTotalYtd: TotalViewModel;
    cryptoTotalYtd: TotalViewModel;

    performanceYtd: TotalViewModel;
}